### DISCUSSION TITLE
Q-select: How to display the full text of a long label?

### DISCUSSION THREAD

**Discussion by davidwbsc** [quasarframework/quasar#13401] [CLI - PWA mode] [answered]
With the q-select I have a label which is longer than the width of the select box so it is truncated with ellipsis.

How can I show the full text of the label? I've tried messing around with some CSS but I think there must be a better way.

**Comment by pdanpdan** [comment]
Make a codepen - it needs some tweaks

**Reply by davidwbsc** [reply]
Here is a codepen showing the issue:

https://codepen.io/davidwbsc/pen/vYdXKgM

**Comment by pdanpdan** [comment]
And how do you want the label to show? On multiple lines?
Or you want the field to expand to the full width of the label? Because you have a width set on the parent.

**Reply by davidwbsc** [reply]
Hi, thanks for your reply. Yes, I think multiple lines would be best. I did try adding a custom class and css like this:

```
.long-label.q-field--auto-height.q-field--labeled .q-field__control-container {
  padding-top: 35px;
}
.long-label .q-field__label {
  text-overflow: initial;
  white-space: initial;
  overflow: initial;
  width: 133%;
}
```

But I think there must be a better way? Also, this doesn't affect the popup, which is outside of the select in the dom, so I can't target it specifically.

**Comment by pdanpdan** [comment, marked as answer]
For multiple lines you should use an external label, or use the label slot, or https://codepen.io/pdanpdan/pen/poaEbGd
